libs/wingsdk/src/cloud/secret.md

```diff
@@ -0,0 +1,79 @@
+---
+title: cloud.Secret 
+id: secret
+description: A built-in resource for securely storing secrets in the cloud.
+keywords: [Wing reference, Wing language, language, Wing sdk, Wing programming language, secrets]
+---
+
+The `cloud.Secret` class represents a secret value (like an API key, certificate, etc.) that is securely stored in the cloud.
+
+Secrets are encrypted at rest and in transit, and are only decrypted when they are used in a task.
+Storing a secret allows you to use the value in different compute tasks while only having to rotate or revoke it in one place.
+
+## Usage
+
+### Creating a secret
+
+```js
+bring cloud;
+
+let secret = new cloud.Secret(
+  name: "my-secret", // optional, defaults to a generated name
```

Comment:
It's worth mentioning that `name` represents the *physical name* of the secret resource.

Reply:
I think I get what you mean, though I'm not sure the best way to explain the concept (I think introducing "physical name" without context might confuse readers)